jekyll/_cci2/workflows.md

```diff
@@ -224,9 +224,9 @@ For example, Scala projects typically require lots of CPU for compilation in the
 
```

Comment:
I think we need an explanation of how workspaces work so people can grok the mental model, maybe

> Each workflow has an associated workspace which can be used to transfer files to downstream jobs as the workflow progresses.
> The workspace is an additive-only store of data. Jobs can persist data to the workspace, this archives the data and creates a new "layer" in an off-container store.
> Downstream jobs can attach the workspace to their container filesystem, this downloads and unpacks each "layer" based on the ordering of the upstream jobs in the workflow graph.